api/api.go

```diff
@@ -111,3 +112,16 @@ func (a *API) createRouter(cfg *config.Config) *fiber.App {
 	protectedAPIRouter.Get("/v1/endpoints/:key/statuses", EndpointStatus)
 	return app
 }
+
+func getReadBufferSize() int {
```

Comment:
Any chance you could add a test that ensures this function returns 4096 by default as well as whatever custom value if the env var is set?